src/UnleashBuilder.php

```diff
@@ -69,6 +70,8 @@ final class UnleashBuilder
 
     private ?string $appName = null;
 
+    private string $connectionId = Uuid::v4();
```

Comment:
I'm open to suggestions of there is a better way of having `connectionId` created only during initialization, and reuse it (alongside with SDK name and version) in both registration and getFeatures calls.